code/modules/gear_presets/survivors/trijent/crashlanding_upp_bar_insert_trijent.dm

```diff
@@ -175,6 +175,61 @@
 	..()
 
 //it's used on all of the above in their spawner.
+
+/datum/equipment_preset/survivor/upp/spacesecurity // CO SURVIVOR
+	name = "Survivor - UPP Ministry of Space Security Officer"
+	flags = EQUIPMENT_PRESET_EXTRA
+	assignment = "MSS Officer"
+	paygrades = list(PAY_SHORT_UO2 = JOB_PLAYTIME_TIER_0)
+	job_title = JOB_UPP_LT_OFFICER
+	faction = FACTION_UPP
+	languages = list(LANGUAGE_RUSSIAN, LANGUAGE_ENGLISH,  LANGUAGE_GERMAN,  LANGUAGE_CHINESE)
+	role_comm_title = "173/RECON MSS"
+	minimap_icon = "upp_lt"
+	minimap_background = "background_upp"
+	idtype = /obj/item/card/id/dogtag/upp
+	skills = /datum/skills/military/survivor/upp_mss
+	access = list(
+		ACCESS_CIVILIAN_PUBLIC,
+	)
+
+/datum/equipment_preset/survivor/upp/spacesecurity/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/veteran/UPP/officer, WEAR_BODY)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/marine/faction/UPP/support, WEAR_JACKET)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine/upp/knife, WEAR_FEET)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/marine/veteran/upp, WEAR_HANDS)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/ivanberet, WEAR_HEAD)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/lightpack/five_slot(new_human), WEAR_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/distress/UPP/recon, WEAR_L_EAR)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/gun/type47, WEAR_WAIST)
+	new_human.equip_to_slot_or_del(new /obj/item/weapon/gun/pistol/t73/leader, WEAR_IN_BELT)
```

Comment:
this has 30% damage over the normal upp pistol (which is already quite good) and accuracy buffs to boot, if you want this pistol specifically as a theme piece you should probably make sure they can only get a mag or two for it.